xbmc/interfaces/legacy/DrmCryptoSession.h

```diff
@@ -294,13 +311,13 @@ namespace XBMCAddon
       /// @brief \python_func{ Verify(macKeyId, message, signature) }
       ///-----------------------------------------------------------------------
       /// \anchor xbmcdrm_Verify
-      /// Verify the validity of a DRM signature of a text message
+      /// Verify the validity of a %DRM signature of a text message.
       ///
-      /// @param      [byte] macKeyId  HMAC key id (provided from a service handshake)
-      /// @param      [byte] message   Message text on which the signature is based
-      /// @param      [byte] signature The signature to verify
+      /// @param     macKeyId  byte - HMAC key id (provided from a service handshake)
+      /// @param     message   byte - Message text on which the signature is based
+      /// @param     signature byte - The signature to verify
       ///
-      /// @return     true when the signature is valid
+      /// @return    true when the signature is valid
       ///
       ///------------------------------------------------------------------------
       /// @python_v18 New function added.
```

Comment:
`/// @python_v19 With python 3 all arguments need to be of type bytearray instead of byte.`

Reply:
done